src/ScriptTools.sol

```diff
@@ -103,20 +110,18 @@ library ScriptTools {
     /**
      * @notice Used to export important contracts to higher level deploy scripts.
      *         Note waiting on Foundry to have better primatives, but roll our own for now.
-     *         Writes contract to out/contract-exports.env
      */
-    function exportContract(string memory name, address addr) internal {
-        vm.writeLine(string(abi.encodePacked(vm.projectRoot(), "/out/contract-exports.env")), string(abi.encodePacked("export FOUNDRY_EXPORT_", name, "=", vm.toString(addr))));
+    function exportContract(string memory json, string memory name, address addr) internal returns (string memory) {
```

Comment:
I'm not seeing something or the `json` param is not being used?

Reply:
I mentioned in Discord, but I refactored a fair bit. Try again should work.